includes/admin/class-wcsatt-admin.php

```diff
@@ -35,6 +35,9 @@ public static function init() {
 		// Creates the panel for configuring subscription options.
 		add_action( 'woocommerce_product_write_panels', __CLASS__ . '::product_write_panel' );
 
+		// Add subscription shipping options on edit product page.
+		add_action( 'woocommerce_product_options_shipping', __CLASS__ . '::subscription_shipping_fields' );
```

Comment:
Given the `WCS_ATT_Admin::subscription_shipping_fields()` function outputs markup identical to `WC_Subscriptions_Admin::subscription_shipping_fields()`, you might as well just use the later and save the duplication. You'll probably need to make sure it will never lead to `WC_Subscriptions_Admin::subscription_shipping_fields()` being called when it's undefined (but hopefully SATT isn't being loaded/executed when Subscriptions isn't defined).